src/LanguageServer/Impl/Implementation/Server.cs

```diff
@@ -50,6 +50,7 @@ public sealed partial class Server : IDisposable {
         private IIndexManager _indexManager;
 
         private InitializeParams _initParams;
+        private bool _initialized;
```

Comment:
Technically, this is racy. I can replace it with something else, but I'm not sure how worthwhile it is for a best-effort bool.

Reply:
shouldn't we default to false?

Reply:
The default boolean is false already.